### DISCUSSION TITLE
Welcome to The Colony Discussions!

### DISCUSSION THREAD

**Discussion by Croquetx** [Croquetx/thecolony#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Croquetx** [comment]
Hi - hope you enjoy looking at the source. It should be buildable using older infrastructure. I particularly like 
https://infinitemac.org/
They have a copy of Think C that should make it possible to build the app. I used that to build the color version of the game as well as Virtus Walkthough.

**Comment by ChipMoody** [comment]
Damn.  I was hoping to land here after a Google search to find someone had adapted this code to run on a current Mac or Win 10!!!  😢

**Reply by yzsocial** [reply]
I started resurrecting this to run in a browser. Decoded the maps and generated a top down view. 
Explored rebuilding the game in Javascript to run in web browsers.

**Reply by Croquetx** [reply]
Replied from wrong account. This is really me.

**Reply by ChipMoody** [reply]
Thanks for the reply!  Was happy to check one of your links posted elsewhere that ran on a web-based emulator!  So...  Happy camper there.  MANY memories coming back from my days working at a humble Apple dealership as a service tech...  💜
